arborist/auth.go

```diff
@@ -337,8 +337,9 @@ func authorizedResources(db *sqlx.DB, request *AuthRequest) ([]ResourceFromQuery
 		if request.Username == "" {
 			return nil, newErrorResponse("missing username in auth request", 400, nil)
 		}
+		// alternative: SELECT DISTINCT * FROM resource WHERE resource.path <@ ARRAY(SELECT resource.path FROM (SELECT usr_policy.policy_id FROM usr JOIN usr_policy ON usr.id = usr_policy.usr_id WHERE usr.name = 'rudyardrichter@uchicago.edu') policies INNER JOIN policy_resource ON policy_resource.policy_id = policies.policy_id INNER JOIN resource ON resource.id = policy_resource.resource_id);
```

Comment:
should we leave this? if we do, you should remove your email

Reply:
I think it's good to leave, I wasn't thorough about checking which would be better

haha yes, good catch. thanks